lms/djangoapps/grades/tasks.py

```diff
@@ -54,8 +55,26 @@ class _BaseTask(PersistOnFailureTask, LoggedTask):  # pylint: disable=abstract-m
     abstract = True
 
 
+@task(base=_BaseTask)
+def compute_all_grades_for_course(**kwargs):
```

Comment:
I agree with you in principle, but the `grades/management/commands/compute_grades.py` code is basically just a "shuffler" for optimizing server load across a batch of multiple courses, versus the (simpler) batching of a single course (which is our use case). 
The tasks as called here, I think, are the best fit for this use case. I did, however, refactor this to use the django admin batch_size overrides, similar to the mgmt command.